tests/customer-resource-custom-coverage-test.js

```diff
@@ -39,7 +37,7 @@ describeApplication('CustomerResourceCustomCoverage', () => {
     });
 
     it('does not display coverage', () => {
-      expect(CustomerResourceCoverage.$root).to.not.exist;
+      expect(CustomerResourceCoverage.exists).to.be.false;
```

Comment:
We could just leave this the same, and we wouldn't need the `isRootPresent` helper: `expect(CustomerResourceCoverage.$root).to.not.exist;`

@wwilsman thoughts?

Reply:
We had to change this because whenever you use `$root` when it doesn't exist it throws an error.